go.mod

```diff
@@ -5,12 +5,7 @@ go 1.14
 // Pin all k8s.io staging repositories to kubernetes-1.15.3.
 // When bumping Kubernetes dependencies, you should update each of these lines
 // to point to the same kubernetes-1.x.y release branch before running update-deps.sh.
-replace (
-	k8s.io/api => k8s.io/api v0.0.0-20190918195907-bd6ac527cfd2
-	k8s.io/apiextensions-apiserver => k8s.io/apiextensions-apiserver v0.0.0-20190918201827-3de75813f604
-	k8s.io/apimachinery => k8s.io/apimachinery v0.0.0-20190817020851-f2f3a405f61d
-	k8s.io/client-go => k8s.io/client-go v0.0.0-20190918200256-06eb1244587a
-)
+replace k8s.io/client-go => k8s.io/client-go v0.0.0-20190918200256-06eb1244587a
```

Comment:
great :)

Reply:
go modules is a hell..i wasted 2 hour try to add cli-runtime and make it compatible with others.